src/ccall.cpp

```diff
@@ -550,14 +550,16 @@ static Value *julia_to_native(
     // pass the address of an alloca'd thing, not a box
     // since those are immutable.
     Value *slot = emit_static_alloca(ctx, to);
+    unsigned align = julia_alignment(jlto);
+    cast<AllocaInst>(slot)->setAlignment(Align(align));
```

Comment:
We should probably make this an argument, since it is pretty much required now by AllocaInst (it will attempt a bad guess if you don't provide it https://llvm.org/doxygen/Instructions_8cpp.html#a35edfaf69cb59d25d92b1e77f0c27530)